seed_data/challenges/computer-science.json

```diff
@@ -21,8 +21,13 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende Informática Básica",
+      "descriptionEs": [
+        "Stanford tiene un excelente currículo de Informática. Este curso interactivo utiliza una versión modificada de JavaScript. Este curso cubrirá rapidamente varias de las ideas.",
+        "Vale recaldar que Harvard también tiene una excelente introducción a Informática en un curso llamado CS50, pero este curso toma más de 100 horas para completar y no utiliza JavaScript.",
```

Comment:
The first phrase should say "Vale *recordar*..." and I would re-write the whole pharagraph as:
"Vale *recordar* que Harvard también tiene *un excelente curso de introducción a la informática llamado CS50*, pero *éste toma más de 100 horas completarlo* y no utiliza JavaScript."